### PR TITLE
EInvoicing: PEPPOL onboarding

### PR DESCRIPTION
**Pull request by beganovich**
This adds onboarding for the PEPPOL standard.

### CODE REVIEW

File: src/pages/settings/e-invoice/peppol/Onboarding.tsx [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+  const form = useFormik({
+    initialValues: {
+      party_name: company?.settings?.name || '',
+      line1: company?.settings?.address1 || '',
+      line2: company?.settings?.address2 || '',
+      city: company?.settings?.city || '',
+      county: company?.settings?.state || '',
+      zip: company?.settings?.postal_code || '',
+      country: company?.settings?.country_id || '',
+    },
```
Comment (turbo124) [inline src/pages/settings/e-invoice/peppol/Onboarding.tsx:272]: we will need tenant_id here which will be company?.company_key

File: src/pages/settings/e-invoice/peppol/Onboarding.tsx [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+    onSubmit: (values, { setSubmitting }) => {
+      toast.processing();
+
+      setErrors(null);
+
+      request('POST', endpoint('/api/v1/einvoice/peppol/setup'), values)
```
Comment (turbo124) [inline src/pages/settings/e-invoice/peppol/Onboarding.tsx:278]: I'm not sure we'll be able to proxy self hosted to the same route due to the guards in place?

**Comment by turbo124** [comment]
@beganovich looks good!

**Comment by beganovich** [comment]
Closed in favour of #2163.

**Pull request closed without merging**